pandas/core/indexes/accessors.py

```diff
@@ -157,6 +157,30 @@ class TimedeltaProperties(Properties):
     """
 
     def to_pytimedelta(self):
+        """
+        Return an array of the Timedeltas in `datetime.timedelta` format with
```

Comment:
Missing single-line summary in the beginning.

Reply:
Added.